apps/elasticsearch/templates/services/app/svc.yml.j2

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: elasticsearch
+    service: elasticsearch
+    version: "{{ elasticsearch_image_tag }}"
+  name: elasticsearch
+  namespace: "{{ project_name }}"
+spec:
+  selector:
+    app: elasticsearch
+    service: elasticsearch
+    type: es-node
+  ports:
+  - name: http
+    port: {{ elasticsearch_api_port }}
```

Comment:
I prefer indenting blocks as follow:

```yaml
key:
  - foo: 1
    bar: 2
```

I would suggest to use `Prettier` on your YAML sources.